Linux/Car_Client/chronoscomm.cpp

```diff
@@ -845,6 +855,65 @@ bool ChronosComm::decodeMsg(quint16 type, quint32 len, QByteArray payload, uint8
         emit monrRx(monr);
     } break;
 
+    case ISO_MSG_TRCM: {
+        chronos_TRCM trcm;
+        VByteArrayLe vb(payload);
+
+        while (!vb.isEmpty()) {
+            uint16_t valueID = vb.vbPopFrontUint16();
+            uint16_t contentLength = vb.vbPopFrontUint16();
+
+            switch(valueID) {
+            case ISO_VALUE_ID_TRIGGER_ID:
+                trcm.triggerID = vb.vbPopFrontUint16();
+                break;
+            case ISO_VALUE_ID_TRIGGER_TYPE:
+                trcm.triggerType = vb.vbPopFrontUint16();
+                break;
+            case ISO_VALUE_ID_TRIGGER_TYPE_PARAM1:
+                trcm.triggerType = vb.vbPopFrontUint32();
+                break;
+            case ISO_VALUE_ID_TRIGGER_TYPE_PARAM2:
+                trcm.triggerType = vb.vbPopFrontUint32();
+                break;
+            case ISO_VALUE_ID_TRIGGER_TYPE_PARAM3:
+                trcm.triggerType = vb.vbPopFrontUint32();
+                break;
+            default:
+                qDebug() << "TRCM: Unknown value id: " << valueID;
+                vb.remove(0, contentLength);
+                break;
+            }
+        }
+        configureTrigger(trcm);
+        qDebug() << "TRCM Rx";
+    } break;
+
+    case ISO_MSG_TREO: {
```

Comment:
The reaction to receiving a TREO on an object should be to reject it, since TREO's are exclusively object -> server directed. So maybe just decode the message and print a warning for now instead of parsing it fully.